src/Symfony/Component/Finder/Tests/FinderTest.php

```diff
@@ -48,6 +48,17 @@ public function testFiles()
         $this->assertIterator($this->toAbsolute(array('foo/bar.tmp', 'test.php', 'test.py', 'foo bar')), $finder->in(self::$tmpDir)->getIterator());
     }
 
+    public function testRemoveTrailingSlash()
+    {
+        $finder = $this->buildFinder();
+
+        $expected = $this->toAbsolute(array('foo/bar.tmp', 'test.php', 'test.py', 'foo bar'));
+        $in = realpath(self::$tmpDir);
+        $in = '//'.realpath(self::$tmpDir).'//';
```

Comment:
Sorry, I removed my previous comment by mistake. It's ok for me if this variable override is fixed :+1:

Reply:
Thanks, fixed